releng/target-platform/target-platform.target

```diff
@@ -11,8 +11,7 @@
   
   Contributors:
       IBM Corporation - initial implementation
--->
-<target includeMode="feature" name="target-platform.target" sequenceNumber="15">
+--><target includeMode="feature" name="target-platform.target" sequenceNumber="15">
```

Comment:
This looks better on separate lines I think.

Reply:
fixed